features/data_ingress_v1.feature

```diff
@@ -11,14 +11,20 @@ Feature: Data Ingress v1
     Given a temporary source directory of files
         | file                   |  fixture                           |
         | pipeline-config.json   |  pipeline_config_basic_valid.json  |
-        | data.xml               |  data_sdmx_valid_1.sdmx            |
+        | data.xml               |  data.xml                          |
```

Comment:
you probably do need a more verbose/descriptive name, remember that over time there'll be many different valid data fixtures in the zip file. I'd find out what dataset it is (cpih? unsure) and include that in the zipped name somehow.